transmission_interface/CMakeLists.txt

```
@@ -87,6 +87,7 @@ target_link_libraries(${PROJECT_NAME}_loader_plugins ${PROJECT_NAME}_loader)
 
 if(CATKIN_ENABLE_TESTING)
   find_package(resource_retriever REQUIRED)
+  include_directories(include ${resource_retriever_INCLUDE_DIRS})
```

Thanks! I definitely missed that in my cleanup of these files... 😬.

We can drop this extra `include`, correct?

```suggestion
  include_directories(${resource_retriever_INCLUDE_DIRS})
```